package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@financial-times/n-sliding-popup",
-  "version": "0.0.0",
+  "version": "4.0.0",
```

Comment:
you don't need to do this, our code will take care of it, best to leave it at `0.0.0` which you can see in other components